docs/src/openapi-v1.yaml

```diff
@@ -980,13 +980,14 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  /experimental/paras:
+  /paras:
     get:
       tags:
       - paras
       summary: |
-        [Experimental - subject to breaking change.] List all registered paras
-        (parathreads & parachains).
+        This endpoint is no longer experimental, please migrate over to using `/paras/*`.
```

Comment:
Maybe just add in release notes?

Reply:
Yea, makes sense. Less clutter in the docs the better.

Reply:
fixed